### PR TITLE
Mongodb task

### PR DESCRIPTION
**Pull request by vyark**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,4 +13,54 @@
...
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.20</version>
```
Comment (fedorchuck) [inline pom.xml]: The good practice to segregate type of dependencies and use variables for storing versions

File: src/main/java/model/Subject.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package model;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
```
Comment (fedorchuck) [inline src/main/java/model/Subject.java:8]: What the difference between Entity and Table?

File: src/main/resources/sql/ddl.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
+drop table if exists student cascade;
```
Comment (fedorchuck) [inline src/main/resources/sql/ddl.sql:1]: good naming of the sql files

**Review by fedorchuck — APPROVED** [review]
Overall it's fine. Good job.
One thing - I'd prefer (and it's a good practice) to add "integration test" to check that it's work as expected.

File: src/main/java/model/Subject.java (hunk 2 above)
Comment (vyark) [reply src/main/java/model/Subject.java:8]: @javax.persistence.Entity: Specifies that the class is an entity. This annotation is applied to the entity class.

@javax.persistence.Table: Specifies the primary table for the annotated entity. Additional tables may be specified using SecondaryTable or SecondaryTables annotation. If no Table annotation is specified for an entity class, the default values apply.

https://34codefactory.wordpress.com/2021/01/08/difference-between-entity-and-table-annotation-code-factory/

**Pull request merged**